### ISSUE TITLE
Progressive Growing of GANs for Improved Quality, Stability, and Variation

### ISSUE THREAD

**Issue by subinium** [subinium/Deep-Papers#29]
- https://arxiv.org/abs/1710.10196
- PGGAN

**Comment by subinium** [comment]
![스크린샷 2021-03-28 오전 1 08 14](https://user-images.githubusercontent.com/16396561/112726682-13ff5680-8f62-11eb-9d71-c117e37fe861.png)

## Progressive Growing of GANs

- Low resolution에서 High resolution을 향해 점진적으로 상향시킴
   - G와 D를 점진적으로 쌓아 올려나가서 균형을 맞추는 것 
   - 점진적으로 쌓지만 freeze하지는 않음
   - Smooth한 학습을 위해 [fade-in](https://arxiv.org/pdf/1505.00387.pdf) 전달 (이전 레이어 저해상도 이미지를 upsampling하여 선형 결합)
- Large-Scale 구조를 먼저 발견하고, 갈수록 디테일한 attribute를 발견  
- 안정적인 훈련, 학습 시간의 단축

## Increasing variation using minibatch standard deviation
 
- GAN은 일반적으로 training data의 부분적인 variation만을 가지고 이미지를 생성
- 이런 문제를 해결하고자 Minibatch Discrimination을 사용 ([Improved Techniques for Training GAN](https://arxiv.org/abs/1606.03498) 참고)

## Normalization in generator and discriminator

- **Equalized learning rate** : weight 
- **Pixelwise feature vector normalization in generator** :

**Comment by subinium** [comment]
## 참고자료

- [김태엽님 모두의 연구소 GAN찮아 3기 발표자료](https://www.slideshare.net/ssuser2a5d00/progressive-growing-of-gans-for-improved-quality-stability-and-variation)
